src/Sylius/Behat/Context/Ui/Shop/LoginContext.php

```diff
@@ -22,23 +27,57 @@
 class LoginContext implements Context
 {
     /**
-     * @var LoginPageInterface
+     * @var CurrentPageResolverInterface
```

Comment:
I guess that we have a convention that `CurrentPageResolverInterface` and `NotificationCheckerInterface ` are the last fields in context class